plugins/hls-fourmolu-plugin/hls-fourmolu-plugin.cabal

```diff
@@ -9,21 +9,27 @@ license:            Apache-2.0
 license-file:       LICENSE
 author:             The Haskell IDE Team
 copyright:          The Haskell IDE Team
+homepage:           https://github.com/haskell/haskell-language-server
+bug-reports:        https://github.com/haskell/haskell-language-server/issues
 maintainer:         alan.zimm@gmail.com
 category:           Development
 build-type:         Simple
 extra-source-files:
   LICENSE
   test/testdata/**/*.hs
 
+source-repository head
+  type:     git
+  location: git://github.com/haskell/haskell-language-server.git
+
 library
   exposed-modules:  Ide.Plugin.Fourmolu
   hs-source-dirs:   src
   ghc-options:      -Wall
   build-depends:
     , base            >=4.12 && <5
     , filepath
-    , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.6
+    , fourmolu        ^>=0.3 || ^>=0.4 || ^>= 0.6 || ^>= 0.7
```

Comment:
This is in the wrong commit, right? Not necessarily actually worth changing now.

Reply:
I am regrettably a complete `git` noob and generally just squash+merge all my stuff 😅

Reply:
Ha, I'm very much in the "git is a wonderful concept with a horrible CLI" camp. Since I finally got around to setting up good editor integration, I haven't looked back.